searchresults/index.html

```diff
@@ -4,17 +4,14 @@
   <title>Flight Booking: Search Results</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no" />
+  <link rel="stylesheet" href="screen.css" />
```

Comment:
remove this line, the css is already listed at line 9 and this file path is wrong